includes/definitions/geist-watchdog.yaml

```diff
@@ -2,9 +2,13 @@ os: geist-watchdog
 text: 'Geist Watchdog'
 type: environment
 icon: geist
+group: geist
 discovery:
-    - sysObjectId:
-        - .1.3.6.1.4.1.21239.
+    -
+        sysObjectId:
+            - .1.3.6.1.4.1.21239.
+        sysDescr_except:
```

Comment:
I'm also now getting errors when run pre-commit about other OS' so it could be my local install :/